Paybutik/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "author": {
+    "name": "Kabal",
+    "id": "269581360316940299"
+  },
+  "service": "Paybutik",
+  "description": {
+    "tr": "Paybutik, müşterilere online ödeme hizmeti sunan bir sağlayıcıdır.",
+    "en": "Paybutik is a provider of online payment service to customers."
+  },
+  "url": "paybutik.net",
+  "version": "1.0",
+  "logo": "https://cdn.serverbutik.com.tr/VyV2Un.png",
+  "thumbnail": "https://cdn.serverbutik.com.tr/vyhBry.png",
```

Comment:
Please upload the images somewhere else so they are not from their own cdn, you could use something like Imgur.